ogre2/src/Ogre2Scene.cc

```diff
@@ -361,6 +362,14 @@ MeshPtr Ogre2Scene::CreateMeshImpl(unsigned int _id, const std::string &_name,
   return this->CreateMeshImpl(_id, _name, descriptor);
 }
 
+//////////////////////////////////////////////////
+EllipsoidPtr Ogre2Scene::CreateEllipsoidImpl(unsigned int _id,
+    const std::string &_name)
+{
+  Ogre2EllipsoidPtr ellipsoid(new Ogre2Ellipsoid);
```

Comment:
I'm curious, why did you create an `Ogre2Ellipsoid` class instead of following the same pattern as sphere, cylinder, box, plane, and using `CreateMeshImpl`?

Reply:
I created an implementation without having a look how the basic shapes were done. I can remove the code in both shapes (capsule and ellipsoid) probably is going to be easy to maintain.

Reply:
@iche033 any thoughts?

Reply:
+1 on using `CreateMeshImpl`. We could add a `unit_ellipsoid` in ign-common and make use of that here in ign-rendering.

Reply:
Update https://github.com/ignitionrobotics/ign-rendering/pull/203/commits/61644e4163a87341bdc308d6b8288e13ac1fbff2